test/unit/version_test.py

```diff
@@ -0,0 +1,207 @@
+import subprocess
```

Comment:
Please move:

- [ ] `test/unit/version_test.py` into `test/unit/util/version_test.py`. 
- [ ] `mock_from_poetry` & `TestTriggerReleaseWithMocking` back into `test/unit/release_test.py`. (These are used to test the `_trigger_release` function in `release.__init__.py`.)